src/OpenTelemetry.ResourceDetectors.AWS/OpenTelemetry.ResourceDetectors.AWS.csproj

```diff
@@ -0,0 +1,33 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <!-- OmniSharp/VS Code requires TargetFrameworks to be in descending order for IntelliSense and analysis. -->
+    <TargetFrameworks>netstandard2.0</TargetFrameworks>
+    <TargetFrameworks Condition="$(OS) == 'Windows_NT'">$(TargetFrameworks);$(NetFrameworkMinimumSupportedVersion)</TargetFrameworks>
+    <Description>OpenTelemetry Extensions - AWS Resource Detectors.</Description>
+    <MinVerTagPrefix>ResourceDetectors.AWS-</MinVerTagPrefix>
+    <Nullable>enable</Nullable>
+    <IncludeSharedExceptionExtensionsSource>true</IncludeSharedExceptionExtensionsSource>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <PackageReference Include="OpenTelemetry" Version="$(OpenTelemetryCoreLatestVersion)" />
+    <PackageReference Include="System.Net.Http" Version="4.3.4" />
+  </ItemGroup>
+
+  <ItemGroup Condition="'$(TargetFramework)' == '$(NetFrameworkMinimumSupportedVersion)'">
```

Comment:
Would this resource detectors not be supported on netframework? If yes, then why not have the project target `net6.0`?

Reply:
In this PR I would like to keep changes minimal. It is just to split packages.

Reply:
More explanation: EKS and ECS Resource Detectors references _linux_ paths. So there is no possibility to support it on .NET Framework.

```
// For ECS
    private const string AWSECSMetadataPath = "/proc/self/cgroup";

//For EKS
     private const string AWSEKSCertificatePath = "/var/run/secrets/kubernetes.io/serviceaccount/ca.crt";
    private const string AWSEKSCredentialPath = "/var/run/secrets/kubernetes.io/serviceaccount/token";
    private const string AWSEKSMetadataFilePath = "/proc/self/cgroup";

```

Reply:
Then this project can just target `net6.0` instead of `netstandard2.0`?

Reply:
In progress. I will create follow up PR and check with @srprash